StandardAnalysis/plugins/EventJetVarProducer.cc

```diff
@@ -125,6 +135,9 @@ EventJetVarProducer::AddVariables (const edm::Event &event, const edm::EventSetu
   edm::Handle<edm::TriggerResults> triggerBits;
   event.getByToken(tokenTriggerBits_, triggerBits);
 
+  TFile* f_jetVeto = TFile::Open("/data/users/mcarrigan/condor/run3Inputs/Summer22EE_23Sep2023_RunEFG_v1.root", "read");
```

Comment:
Will this file live in your condor directory from now on or should be include this file in the repository?

Reply:
I just don't want it to be a part of every .tar.gz file that we make when running so I planned to save it in /data and have everyone pull from there. Similar to what was done in the past for some files

Reply:
Maybe it can be put inside `StandardAnalysis/data`?